docs/Development Concepts/6.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Security Features
+---
+
+Each protocol layer of Waku v2 provides a distinct service and is associated with a separate set of security features and concerns.
+Therefore, the overall security of Waku v2 depends on how the different layers are utilized.
+
+Some of the security features of Waku v2 are:
+
+### Pseudonymity
```

Comment:
I assume this is all taken from the specs? Tagging @kaiserd to ensure our statements are true.